tests/search/inconsistent_replicas/updates_to_inconsistent_buckets.rb

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'search_test'
+
+class UpdatesToInconsistentBucketsTest < SearchTest
+
+  def setup
+    set_owner('vekterli')
+    deploy_app(make_app())
+    start
+  end
+
+  def teardown
+    stop
+  end
+
+  def make_app
+    SearchApp.new.sd(SEARCH_DATA + 'music.sd').
+      cluster_name('storage').
+      num_parts(2).redundancy(2).ready_copies(2).
+      enable_http_gateway.
+      storage(StorageCluster.new('storage', 2).distribution_bits(8)).
+      config(ConfigOverride.new('vespa.config.content.core.stor-distributormanager').
+             add('merge_operations_disabled', true).
+             add('restart_with_fast_update_path_if_all_get_timestamps_are_consistent', true))
+  end
+
+  def api_http_post(path, content)
+    vespa.document_api_v1.http_post(path, content)
+  end
+
+  def api_http_put(path, content)
+    vespa.document_api_v1.http_put(path, content)
+  end
+
+  def api_http_get(path)
+    response = vespa.document_api_v1.http_get(path)
+    vespa.document_api_v1.assert_response_ok(response)
+    JSON.parse(response.body)['fields']
+  end
+
+  def feed_doc_with_field_value(title:)
+    # Also add a second field that updates won't touch so that we can detect if
+    # a 'create: true' update erroneously resets the state on any replica.
+    doc = { 'fields' => { 'title' => title, 'artist' => 'cool dude' } }
+    response = api_http_post("/document/v1/storage_test/music/number/1/foo", doc.to_json)
```

Comment:
The DocumentApiV1 class has some helper functions (put, get, remove) that constructs the path for you. Consider using get put() function instead.

Reply:
Done